plugins/excel/core/src/main/resources/org/pentaho/di/trans/steps/excelwriter/messages/messages_en_US.properties

```diff
@@ -166,6 +168,9 @@ ExcelWriter.Injection.START_WRITING_AT_SHEET_END=Decide if you want the output t
 ExcelWriter.Injection.OFFSET_BY_ROWS=Specify any non-zero number to cause the step to move this amount of rows down (positive numbers) or up (negative numbers) before writing rows.
 ExcelWriter.Injection.BEGIN_BY_WRITING_EMPTY_LINES=Specify any positive number to cause the step to leave this amount of rows empty.
 ExcelWriter.Injection.OMIT_HEADER=Decide if you want to include a header when writing to an existing spreadsheet.
+ExcelWriter.Injection.EXTEND_DATA_VALIDATION=Data validations affecting the first row will be extended to all data
+ExcelWriter.Injection.CREATE_PARENT_FOLDERS=If folders missing in the target file''s path should be created.
+ExcelWriter.Injection.RETAIN_NULL_VALUES=If checked, NULL values will be preserved in your output. Otherwise, NULL values will be replaced with an empty string.
```

Comment:
The defaults are weird for this. Default above is true, but this implies that if the user doesn't act, the default is false.

Reply:
this was copied from the checkbox description, which was already there. What phrasing would you suggest here?

Reply:
You could instead change the default in the java code. (from true to false for the default) 

OH! use case: upgrade! the behavior of old ktrs that use this step shouldn't change. So the default in the java code should be whatever the old behavior was for this step.

Reply:
AH, "True" is the old behavior, so the current defaults are correct.